WebRoot/skins/_base/base4/skin.properties

```diff
@@ -881,7 +882,7 @@ ListItem-multipleDragProxy      = font-weight:bold; text-align: center; backgrou
 #   OVERVIEW TREE
 #####################
 
-OverviewTreeContainer       = @AppBg@; border-top: 1px solid @PanelBorderColor@;
+OverviewTreeContainer       = @AppBg@; border:1px solid @PanelBorderColor@; border-left:0; border-right:0;
```

Comment:
Anything will work. I finds it more readable.
Let me know if you really feels need to change this as per your suggestions.  

Thanks